.pre-commit-config.yaml

```diff
@@ -16,6 +16,7 @@ repos:
     rev: v1.10.0
     hooks:
       - id: mypy
+        additional_dependencies: ["types-requests"]
```

Comment:
This addition was required to fix a `mypy` error `Library stubs not installed for "requests"  [import-untyped]`